USFMConverter/USFMConverter.csproj

```diff
@@ -5,13 +5,14 @@
     <Nullable>enable</Nullable>
     <ApplicationIcon>UI\Assets\converter_logo_inverted_icon.ico</ApplicationIcon>
     <UseAppHost>true</UseAppHost>
+    <AssemblyVersion>1.1.1</AssemblyVersion>
 <!--    MacOS bundle information begin-->
     <CFBundleSignature>????</CFBundleSignature>
     <CFBundleName>USFMConverter</CFBundleName> <!-- Also defines .app file name -->
     <CFBundleDisplayName>USFMConverter</CFBundleDisplayName>
     <CFBundleIdentifier>org.wycliffeassociates.USFMconverter</CFBundleIdentifier>
-    <CFBundleVersion>1.1.1</CFBundleVersion>
-    <CFBundleShortVersionString>1.1.1</CFBundleShortVersionString>
+    <CFBundleVersion>$(AssemblyVersion)</CFBundleVersion>
+    <CFBundleShortVersionString>$(AssemblyVersion)</CFBundleShortVersionString>
```

Comment:
Nice!

Reply:
MSBuild variable expansion is really nice